packages/components/src/components/navbar/Usage.mdx

```diff
@@ -0,0 +1,179 @@
+
+
+**Expanded Submenu (3)**\
+A text link with a chevron-up icon which hints that a mega-menu or a simple drop-down menu is collapsible. This item does not lead to a different page.
+
+
+**More Submenu (4)**\
+Due to space or smaller breakpoints it is possible to use a "More" text item which indicates there are more options in the 1st level navigation. Those text items should be placed inside a submenu component.
+
+
+### Secondary Navigation Items
+<img src={HeaderSecondaryNavigationItemsExample} alt="Anatomy Header Main Items" width="50%" />
+
+
+**Search (1)**\
+Always positioned at the **leftmost** position. To underline its functionality we recommend to add a "Search" label to the icon.
+
+
+**Other (2)**\
+Special or less important functions, which you can't fit into the menu, can be positioned right hand next to the search.
+
+
+**Notification (3)**\
+The notification icon is right aligned. But cart (4), profile (5) and button (6) have priority regarding the alignment.
+
+
+**Cart (4)**\
+For product websites with shopping-like functions, the cart icon is a critical item. It should be placed **third from the right**.
+
+
+**Profile (5)**\
+The profile icon is a key item of many websites, so it should be easily accessible. Place it **second from the right**.
+
+
+**Button (6)**\
+Depending on the context, the button can be used for a variety of functions such as initiating an action. Place it **rightmost** for easy accessibility.
+
+
+### Placement by priority
+The table below gives you an overview and guidance how to prioritize your secondary navigation items.
+
+| Breakpoint | Logo      | Application name | Main navigation | Secondary navigation | Search field |
+|------------|-----------|------------------|-----------------|----------------------|--------------|
+| ≥1440px    | Mandatory | High             | High            | High                 | High         |
+| ≥1200px    | Mandatory | High             | High            | High                 | Low          |
+| ≥1024px    | Mandatory | High             | n/c             | High                 | n/c          |
+| ≥720px     | Mandatory | Low              | n/c             | High                 | n/c          |
+| <720px     | Mandatory | Low              | n/c             | High                 | n/c          |
+
+## Breakpoints
+<img src={HeaderOverviewBreakpoints} alt="Header Breakpoints" />
+
+**≥1440px**\
+A large header with lots of space and the possibility to display each element.
+
+
+**≥1200px**\
+At this width you can decide by priority what to display. 
+
+
+**≥1024px**\
+At this width you should change your visible search bar into an icon with a label localized in the secondary navigation. Furthermore it is possible to move less important text links into a collapsible "More" text link.
+
+
+**≥720px**\
+At this width the header is simplified even further. Text links are moved into the burger menu.
+
+
+**<720px**\
```

Comment:
Storybook wasnt able to render the "<" symbol. It needs to be escaped like this: "\<"